.github/actions/aws-ci/action.yml

```diff
@@ -47,8 +54,9 @@ runs:
       working-directory: ./tests
       env:
         USE_K8S: "false"
+        KEYWORD: ${{ inputs.pytest-keyword-expression }}
       shell: bash
-      run: poetry run pytest -k "aws" --alluredir=./allure/results/ --clean-alluredir
+      run: poetry run pytest -k "${{ env.KEYWORD }}" --alluredir=./allure/results/ --clean-alluredir
```

Comment:
```suggestion
      run: poetry run pytest -k "$KEYWORD" --alluredir=./allure/results/ --clean-alluredir
```

Reply:
Resolved by having a separate action for asset inventory.